cmd/rbacgen/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/omc-college/management-system/pkg/rbac/models"
+	"github.com/omc-college/management-system/pkg/rbac/openapi"
```

Comment:
lets split imports to three sections:
1. std lib
2. external deps
3. projects package  imports